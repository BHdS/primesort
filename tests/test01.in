590
830
200
55
3
1
2
1
500
900
500
30
20
50
90
-1
