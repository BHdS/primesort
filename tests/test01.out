900
830
590
500
500
200
90
55
50
30
20
1
1
